### PR TITLE
Add PWM configuration

### PR DESCRIPTION
**Pull request by mariuscrsn**

### CODE REVIEW

File: firmware/Sources/HAL/HAL_SYS/hal_sys.c [hunk 1]
```diff
@@ -8,7 +8,7 @@
 /**********************************************************************************/
 #include "epc_conf.h"
 #include "epc_st_err.h" //Import EPC_ST_ERR_COUNTER
-#include "stm32f3xx_hal.h"
+#include "main.h"
```
Comment (pastorpflores) [inline firmware/Sources/HAL/HAL_SYS/hal_sys.c:11]: change to `#include "stm32f3xx_hal.h"`

File: firmware/Sources/Test/HAL/hal_sts_test.c [hunk 2]
```diff
@@ -6,6 +6,7 @@
 /**********************************************************************************/
 /*                  Include common and project definition header                  */
 /**********************************************************************************/
+#include "main.h"
```
Comment (pastorpflores) [inline firmware/Sources/Test/HAL/hal_sts_test.c:9]: change to `#include "stm32f3xx_hal.h"`

**Review by pastorpflores — CHANGES_REQUESTED** [review]

File: firmware/Sources/HAL/HAL_SYS/hal_sys.c (hunk 1 above)
Comment (mariuscrsn) [reply firmware/Sources/HAL/HAL_SYS/hal_sys.c]: Done

File: firmware/Sources/Test/HAL/hal_sts_test.c (hunk 2 above)
Comment (mariuscrsn) [reply firmware/Sources/Test/HAL/hal_sts_test.c]: Done

**Review by pastorpflores — APPROVED** [review]

**Pull request merged**